src/main/scala/no/ndla/searchapi/model/taxonomy/TaxonomyBundle.scala

```diff
@@ -15,20 +15,25 @@ case class TaxonomyBundle(
     resourceTypes: List[ResourceType],
     resources: List[Resource],
     subjectTopicConnections: List[SubjectTopicConnection],
-    subjects: List[Resource],
+    subjects: List[TaxSubject],
     topicFilterConnections: List[TopicFilterConnection],
     topicResourceConnections: List[TopicResourceConnection],
     topicSubtopicConnections: List[TopicSubtopicConnection],
     topicResourceTypeConnections: List[TopicResourceTypeConnection],
-    topics: List[Resource]
+    topics: List[Topic]
 ) {
 
-  def getResourceTopics(resource: Resource): List[Resource] = {
+  def getResourceTopics(resource: Resource): List[Topic] = {
     val tc = topicResourceConnections.filter(_.resourceId == resource.id)
     topics.filter(topic => tc.map(_.topicid).contains(topic.id))
   }
 
-  def getObject(id: String): Option[Resource] = {
+  def getSubject(path: String): Option[TaxSubject] = {
+    val subject = path.split('/')(1)
+    subjects.find(_.id == s"urn:$subject")
```

Comment:
Hadde likt denne bedre om den ikke kunne kaste exceptions.
Hva med noe sånt? :)
```suggestion
    val subject = path.split('/').lift(1)
    subject.flatMap(s => subjects.find(_.id == s"urn:$s"))
```